src/rascal/representations/calculator_spherical_expansion.hh

```diff
@@ -2158,26 +2158,26 @@ namespace rascal {
               << "center_atoms_mask should not mask atoms.";
       throw std::runtime_error(err_str.str());
     }
-    auto manager_root = extract_underlying_manager<0>(manager);
-    auto cell_length = manager_root->get_cell_length();
-    auto pbc = manager_root->get_periodic_boundary_conditions();
-    bool is_cutoff_too_large{false};
-    for (size_t i_dim{0}; i_dim < ThreeD; ++i_dim) {
-      if (pbc[i_dim]) {
-        if (cell_length[i_dim] < 2. * this->interaction_cutoff) {
-          is_cutoff_too_large = true;
-        }
-      }
-    }
-    if (IsHalfNL and is_cutoff_too_large) {
-      std::stringstream err_str{};
-      err_str << "Half neighbor list should only be used when the diameter of "
-              << "the spherical expansion is smaller than the unit cell "
-              << "in periodic directions: "
-              << "[" << cell_length.transpose() << "] > "
-              << 2 * this->interaction_cutoff;
-      throw std::runtime_error(err_str.str());
-    }
+    //auto manager_root = extract_underlying_manager<0>(manager);
+    //auto cell_length = manager_root->get_cell_length();
+    //auto pbc = manager_root->get_periodic_boundary_conditions();
```

Comment:
Is this what you mean with the sentence below?

> disabled some checks which require cell information which we do not have in the case of lammps

The periodicity of the cell should be available somewhere in lammps, it is set with the [boundary](https://docs.lammps.org/boundary.html) command in the input.

The cell lengths can be recomputed if we want from LAMMPS `xlo xhi` and friends.

Reply:
Yes this part is meant. Also in stress one line is commented out. Yes it should be somewhere available, it is not so urgent at the moment.